wgengine/userspace.go

```diff
@@ -136,6 +136,7 @@ type userspaceEngine struct {
 	networkMapCallbacks map[*someHandle]NetworkMapCallback
 	tsIPByIPPort        map[netaddr.IPPort]netaddr.IP          // allows registration of IP:ports as belonging to a certain Tailscale IP for whois lookups
 	pongCallback        map[[8]byte]func(packet.TSMPPongReply) // for TSMP pong responses
+	icmpEchoCallback    map[uint32]func()                      // for ICMP echo responses
```

Comment:
document map key

I guess pongCallback has the same problem

Reply:
I'll do both!